CODEOWNERS

```diff
@@ -15,3 +15,4 @@ requirements-dev.txt @microsoft/onnxruntime-admin
 requirements-doc.txt @microsoft/onnxruntime-admin
 requirements-training.txt @microsoft/onnxruntime-admin
 requirements.txt.in @microsoft/onnxruntime-admin
+/cgmanifests/cgmanifest.json @microsoft/onnxruntime-admin
```

Comment:
should cgmanifests/generated/cgmanifest.json be included too? it has the submodule entries.

Reply:
Thanks, added this file too